Classes/Auth/TMOAuth.m

```diff
@@ -37,6 +37,34 @@ + (NSString *)headerForURL:(NSURL *)URL method:(NSString *)method postParameters
     return auth.headerString;
 }
 
++ (NSURL *)signUrlWithQueryComponent:(NSURL *)URL method:(NSString *)method postParameters:(NSDictionary *)postParameters
+                               nonce:(NSString *)nonce consumerKey:(NSString *)consumerKey consumerSecret:(NSString *)consumerSecret
+                               token:(NSString *)token tokenSecret:(NSString *)tokenSecret timestamp:(NSString *)timestamp {
+    NSMutableDictionary *oauthParameters = [[NSMutableDictionary alloc] initWithDictionary:@{
```

Comment:
A lot of this is copied from the other method, is there a way we can factor this in a DRY way?